jerry-debugger/src/lib/cdt-proxy.ts

```diff
@@ -36,13 +44,62 @@ export class ChromeDevToolsProxyServer {
       ws.send('hello');
     });
 
+    server.listen(SERVER_PORT);
+
+    // TODO: move callback out to standalone method
+    server.on('request', function (request: http.IncomingMessage, response: http.ServerResponse) {
+      var url = request.url || '';
+      if (url == '/json') {
+        url = '/json/list';
+      }
+
+      if (request.method != "GET" || url.slice(0, 6) != '/json/') {
+        return;
+      }
+
+      var cmd = url.slice(6);
+      var result = undefined;
+      switch (cmd) {
+        case 'version':
+          result = {
+            'Browser': 'jerry-debugger/v0.0.1',
+            'Protocol-Version': '1.1'
+          };
+          break;
+        case 'list':
+          var urlfrag = `127.0.0.1:${SERVER_PORT}/${SERVER_UUID}`;
+          result = [{
+            'description': 'zephyr.js instance',
+            'devtoolsFrontendUrl': `https://chrome-devtools-frontend.appspot.com/serve_file/@60cd6e859b9f557d2312f5bf532f6aec5f284980/inspector.html?experiments=true&v8only=true&ws=${urlfrag}`,
+            'id': SERVER_UUID,
+            // FIXME: probably not quite right, can include ../.. etc.
+            'title': options.jsfile || 'untitled.js',
+            'type': 'node',
+            'url': '',
```

Comment:
Do you know what this `url` is used for?

Reply:
Just from watching the wire protocol it's a file:/// URL to the 'jsfile'. I'll include a note to that effect my next push.